Examples/Io/Root/src/RootVertexPerformanceWriter.cpp

```diff
@@ -238,50 +238,56 @@ ActsExamples::ProcessCode ActsExamples::RootVertexPerformanceWriter::writeT(
     }
   } else {
     // get active tips
-    const auto& trajectories =
+    const auto& inputTrajectories =
         ctx.eventStore.get<TrajectoriesContainer>(m_cfg.inputTrajectories);
-    const auto& trackTips =
-        ctx.eventStore.get<std::vector<std::pair<size_t, size_t>>>(
-            m_cfg.inputTrackParametersTips);
 
     std::vector<ParticleHitCount> particleHitCounts;
 
     using HitParticlesMap = IndexMultimap<ActsFatras::Barcode>;
     const auto& hitParticlesMap =
         ctx.eventStore.get<HitParticlesMap>(m_cfg.inputMeasurementParticlesMap);
 
-    for (size_t i = 0; i < trackParameters.size(); i++) {
-      auto& [iTraj, tip] = trackTips[i];
-      const auto& traj = trajectories[iTraj];
-      identifyContributingParticles(hitParticlesMap, traj, tip,
-                                    particleHitCounts);
-      ActsFatras::Barcode majorityParticleId =
-          particleHitCounts.front().particleId;
-      size_t nMajorityHits = particleHitCounts.front().hitCount;
-
-      auto trajState = Acts::MultiTrajectoryHelpers::trajectoryState(
-          traj.multiTrajectory(), tip);
-
-      if (nMajorityHits * 1. / trajState.nMeasurements <
-          m_cfg.truthMatchProbMin) {
-        continue;
-      }
+    for (const auto& trajectories : inputTrajectories) {
```

Comment:
Maybe `trajectory` (singular) for clarity?

Reply:
this is generally very confusing to me... `inputTrajectories` is a trajector**ies** container. so even the elements are multiple trajector**ies**. this is not well empathized in the examples framework so I would stick with the current "convention"

Reply:
I would agree that this is not optimal, and there is no obvious better solution than this, so lets keep it like this.